### DISCUSSION TITLE
Can't run Chrome as browser for E2E testing - other browsers work fine.  Any ideas?

### DISCUSSION THREAD

**Discussion by rkobetich** [cypress-io/cypress#30666] [Questions and Help]
Can't run Chrome as browser for E2E testing - other browsers work fine.  Any ideas?

Error message - Whoops, we can't run your tests.

This browser was not launched through Cypress. Tests cannot run.

**Comment by MikeMcC399** [comment]
@rkobetich 

If you are running under Windows, then you may find that the following applies to your problem:

https://docs.cypress.io/app/references/error-messages#Cypress-detected-policy-settings-on-your-computer-that-may-cause-issues

**Comment by rkobetich** [comment]
thanks
